src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+
+import { Dropdown as CoreDropdown } from 'wix-ui-core/dropdown';
+
+import { DropdownOption } from '../Dropdown/DropdownOption';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import styles from './FloatingDropdown.st.css';
+import { FloatingDropdownBase } from './FloatingDropdownBase';
+import { DATA_HOOKS } from './constants';
+import { FloatingDropdownOptionProps } from './FloatingDropdownOption';
+
+export interface FloatingDropdownProps {
+  /** Defines a string value that labels the current element. Optional. */
+  'aria-label'?: string;
```

Comment:
@jonathanadler I was not 100% sure about ARIA labels, so just copied ones used in `<Dropdown />`

Reply:
👍